prebuild/xmlsec-1-3-14-deps

```diff
@@ -0,0 +1,124 @@
+#!/usr/bin/env python3
```

Comment:
this is pretty annoying -- lxml (python) and xmlsec (python) and libxml2 and libxmlsec1 all need to match versions or `xmlsec` won't import